docs/vscode.md

```diff
@@ -41,7 +41,7 @@ You can create a `.vscode/c_cpp_properties.json` file with `C/C++: Edit Configur
 {
     "configurations": [
         {
-            "name": "Linux",
+            "name": "OoT64 gc-eu-mq-dbg",
             "compilerPath": "${default}", // Needs to not be "" for -m32 to work
```

Comment:
no need for OoT64 imo, but meh

Reply:
yeah idk
personally I locally have another configuration I switch to for native tools (like mkdmadata and co), which I'd recommend to anyone, so it's more to separate "cross compile for N64" from "compile native code"
I can change it to "N64 gc-eu-mq-dbg", "N64 oot-gc-eu-mq-dbg" ?

Reply:
sure either of those are fine with me. i guess i slightly prefer without "oot" but doesnt matter